### PR TITLE
[JBPM-9344] Maven project loader requires solving pom with immutable spring boot images.

### PR DESCRIPTION
**Pull request by elguardian**
There is no need to resolve artifacts anymore when building kie module. There should be already package.
https://issues.redhat.com/browse/JBPM-9344

### CODE REVIEW

File: kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/InJarArtifactResolver.java [hunk 1]
```diff
@@ -26,8 +26,10 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
+import org.apache.maven.model.Dependency;
```
Comment (afalhambra) [inline kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/InJarArtifactResolver.java:29]: Unused import. Please remove

**Review by afalhambra — CHANGES_REQUESTED** [review]
Very small change requested. Otherwise looks great to me

File: kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/InJarArtifactResolver.java (hunk 1 above)
Comment (elguardian) [reply kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/InJarArtifactResolver.java]: done

**Review by afalhambra — APPROVED** [review]
Very nice catch! well done

**Pull request merged**